zebra-network/src/peer/error.rs

```diff
@@ -28,6 +28,9 @@ pub enum PeerError {
     /// The remote peer closed the connection.
     #[error("Peer closed connection")]
     ConnectionClosed,
+    /// The local client closed the connection.
+    #[error("Internal client dropped connection")]
+    ConnectionDropped,
```

Comment:
I think `client` could be confusing here, because we also have a `zebra-client` crate. But it's a hard one to name.

Maybe we want to say something like:
```suggestion
    /// The local node closed the connection.
    #[error("Internal node dropped connection")]
    ConnectionDropped,
```

Reply:
Deleted by 4c75fa2